ledger/src/blockstore_processor.rs

```diff
@@ -934,8 +927,7 @@ fn load_frozen_forks(
             }
```

Comment:
how about renaming this fn to `load_fork_tip_banks()` because it can return unfrozen banks now?

Reply:
also, for `solana-ledger-tool`, I think it's better to retain the old behavior somehow?